cmake/modules/AddSwift.cmake

```diff
@@ -2437,3 +2445,9 @@ macro(add_swift_tool_symlink name dest component)
   add_llvm_tool_symlink(${name} ${dest} ALWAYS_GENERATE)
   llvm_install_symlink(${name} ${dest} ALWAYS_GENERATE COMPONENT ${component})
 endmacro()
+
+function(add_swift_install_targets target component)
```

Comment:
Is there a reason for creating this really?  It seems that we could just directly use `add_llvm_install_targets`.